src/Bar.js

```diff
@@ -117,7 +117,7 @@ export default class BarChart extends Component {
     let textStyle = fontAdapt(options.axisX.label)
 
     let lines = chart.curves.map(function (c, i) {
-      let color = this.color(i % 3)
+      let color = _.isArray(this.props.options.color) ? this.props.options.color[i] : this.color(i % 3)
```

Comment:
If the number of items in the color array is less than the number of items, you will get this error:

```
null is not an object (evaluating 'c.r')
```

Similarly, if you have a data structure formed as you do in the example app where you have multiple arrays in the data set like:

```
    let data = [
      [{
        "v": 49,
        "name": "apple"
      }, {
        "v": 42,
        "name": "apple"
      }],
      [{
        "v": 69,
        "name": "banana"
      }, {
        "v": 62,
        "name": "banana"
      }],
      [{
        "v": 29,
        "name": "grape"
      }, {
        "v": 15,
        "name": "grape"
      }]
    ]
```

you will get the same "null is not an object (evaluating 'c.r')" error if you don't repeat the color definitions in the array like this:

```
color: ['#2980B9', '#3d6929', '#9a388b','#2980B9', '#3d6929', '#9a388b'],
```

Can you address both of these scenarios?